fv3core/stencils/sim1_solver.py

```diff
@@ -87,7 +79,7 @@ def sim1_solver(
             gam = aa / bet[0, 0, -1]
             bet = dm - (aa + p1 + aa * gam)
             w = (
-                dm * w1 + dt * (pp[0, 0, 1] - pp) - p1 * wsr_top - aa * w[0, 0, -1]
+                dm * w1 + dt * (pp[0, 0, 1] - pp) - p1 * wsr[0, 0] - aa * w[0, 0, -1]
```

Comment:
This is more for my information- is the [0, 0] there to make it concise that wsr is 2d?

Reply:
Yeah exactly, it is not required, but helps the readability to point out that it is a 2d field